aiohttp/web_response.py

```diff
@@ -54,6 +54,8 @@ def __init__(self, *, status=200, reason=None, headers=None):
 
         if headers is not None:
             self._headers = CIMultiDict(headers)
+            if None in headers.values():
```

Comment:
This must be slow.
@asvetlov wouldn't it be more efficient to put the check in multidict around `__setitem__()`?